JavaScript/JavaScriptSDK.Tests/CheckinTests/Logging.tests.ts

```diff
@@ -73,17 +76,17 @@ class LoggingTests extends TestClass {
                 InternalLogging.verboseLogging = () => true;
 
                 // act
-                InternalLogging.throwInternalNonUserActionable(Microsoft.ApplicationInsights.LoggingSeverity.WARNING, new InternalLoggingMessage("error!"));
-                InternalLogging.throwInternalUserActionable(Microsoft.ApplicationInsights.LoggingSeverity.WARNING, new InternalLoggingMessage("error!"));
-                InternalLogging.throwInternalNonUserActionable(Microsoft.ApplicationInsights.LoggingSeverity.CRITICAL, new InternalLoggingMessage("error!"));
-                InternalLogging.throwInternalUserActionable(Microsoft.ApplicationInsights.LoggingSeverity.CRITICAL, new InternalLoggingMessage("error!"));
+                InternalLogging.throwInternalNonUserActionable(Microsoft.ApplicationInsights.LoggingSeverity.WARNING, new InternalLoggingMessage(3, "error!"));
```

Comment:
Numbers "3", "4", etc look confusing. Does that matter for the test? if yes, can you do global 
``var i = 0;``
 and then pass 
``++i``
as the 1st arg in all tests?